tests/simulator_core/test_thing_loader.py

```diff
@@ -0,0 +1,165 @@
+from pathlib import Path
+import pytest
+import simulator.simulator_core.thing_parser as thing_parser
+import tests.simulator_core.test_defs as defs
+import json
+
+ACCEPTABLE_DIFFERENCE = 0.000000000000001
+
+test_file_path = str(Path(__file__).parent.parent)
+valid_raw_file = test_file_path + "/test_files/valid_raw_config.json"
+invalid_raw_json_id = test_file_path + "/test_files/invalid_raw_config_no_id.json"
```

Comment:
This line don't follow the 88 character rule, make sure that your linter is working properly. This occurs throughout the code.